.github/workflows/js_wallet_publish.yml

```diff
@@ -6,39 +6,43 @@ on:
       - walletjs-*
 
 jobs:
-  initial_release:
-    name: Publish wasm package
-    runs-on: ubuntu-latest
-    outputs:
-      upload_url: ${{ steps.create_release.outputs.upload_url }}
-      version: ${{ steps.get_version.outputs.version }}
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
+  # initial_release:
+  #   name: Publish wasm package
+  #   runs-on: ubuntu-latest
+  #   outputs:
+  #     upload_url: ${{ steps.create_release.outputs.upload_url }}
+  #     version: ${{ steps.get_version.outputs.version }}
+  #   steps:
+  #     - name: Checkout code
+  #       uses: actions/checkout@v3
 
-      - name: Create Release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ github.ref }}
-          release_name: Release ${{ github.ref }}
-          draft: true
-          prerelease: false
+  #     - name: Create Release
+  #       id: create_release
+  #       uses: actions/create-release@v1
+  #       env:
+  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  #       with:
+  #         tag_name: ${{ github.ref }}
+  #         release_name: Release ${{ github.ref }}
+  #         draft: true
+  #         prerelease: false
 
-      - name: Set version output
-        id: get_version
-        run: echo ::set-output name=version::``${GITHUB_REF#refs/tags/}``
+  #     - name: Set version output
+  #       id: get_version
+  #       run: echo ::set-output name=version::``${GITHUB_REF#refs/tags/}``
 
-  release_wasm:
-    name: Release wasm assets
-    needs: initial_release
+  release_wallet_js:
+    name: Release wallet js assets
+    # needs: initial_release
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        target: [nodejs, web, bundler]
+        target: [
+                nodejs, 
+                # web,
```

Comment:
Should we uncomment `web` and `bundler`?

Reply:
At this moment it is not possible to publish a few versions of packages under the same rpm package registry (at least at that way which we are doing at this moment), so only one job can finish successfully. I would let at this moment only `nodejs` version and figure out how it is better to publish a few later.